package.json

```diff
@@ -3,11 +3,11 @@
   "version": "0.0.0-development",
   "description": "A drop-in module that adds autoUpdating capabilities to Electron apps",
   "repository": "https://github.com/electron/update-electron-app",
-  "main": "index.js",
-  "types": "index.d.ts",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
   "license": "MIT",
   "dependencies": {
-    "electron-is-dev": "^0.3.0",
+    "electron-is-dev": "^2.0.0",
```

Comment:
question: Do we still need `electron-is-dev`, or can we just use `app.isPackaged`  directly nowadays? I looked at the package's code and it does a couple other safety checks that I'm not sure warrant adding a whole new package to the dependency tree.

```js
'use strict';
const electron = require('electron');

if (typeof electron === 'string') {
	throw new TypeError('Not running in an Electron environment!');
}

const isEnvSet = 'ELECTRON_IS_DEV' in process.env;
const getFromEnv = Number.parseInt(process.env.ELECTRON_IS_DEV, 10) === 1;

module.exports = isEnvSet ? getFromEnv : !electron.app.isPackaged;
```

The only thing I can see is that there's a custom `ELECTRON_IS_DEV` environment variable case that would need to be supported to avoid breaking existing use cases.